config/locales/en.yml

```diff
@@ -1414,20 +1414,14 @@ en:
           legend: "Connected apps"
           affiliate_sales_data:
             title: "INRAE / UFC QUE CHOISIR Research"
-            tagline: "Allow this research project to access your orders data anonymously"
+            tagline: "Allow this research project to access your orders data anonymously."
             enable: "Allow data sharing"
             disable: "Stop sharing"
             loading: "Loading"
             need_to_be_manager: "Only managers can connect apps."
-            description_html: |
-              <p>
-              INRAE and UFC QUE CHOISIR are teaming up to study food prices in short food systems and compare them with prices in the supermarket, for a given set of products. The data that is used by INRAE is mixed with data coming from other short food chain platforms in France. No individual product prices will be publicly disclosed through this project.
-              </p>
-              <p>
-              <a href="https://apropos.coopcircuits.fr/"
-              target="_blank"><b>Learn more about this research project</b>
-              <i class="icon-external-link"></i></a>
-              </p>
+            description: "INRAE and UFC QUE CHOISIR are teaming up to study food prices in short food systems and compare them with prices in the supermarket, for a given set of products. The data that is used by INRAE is mixed with data coming from other short food chain platforms in France. No individual product prices will be publicly disclosed through this project."
+            link_label_html: "Learn more about this research project <i class=\"icon-external-link\"></i>"
```

Comment:
While we're at it, we could move ` <i class=\"icon-external-link\"></i>` to the HAML also. 

Or perhaps even better, add a style rule so that external links are automatically styled with an icon. 

But this is fine. Translators can just work around it.

Reply:
> Or perhaps even better, add a style rule so that external links are automatically styled with an icon.

I've been thinking about that, too. Shall we create a good-first-issue for that?

Reply:
We'd want to define exactly where the style rule applies. For example, we don't want it to appear in the navigation menu because 'about.openfoodnetwork.org' is a different domain.

I think it should actually based on the `target`, not the domain name. So only show if `target="_blank"`. IE:
```css
a[target="_blank"]::after {
  /*show the icon.. */
}
```
What do you think?

This could probably be applied across the entire frontend and backend, but we would still need to test that it doesn't appear wrong in any circumstances, which is a lot of testing.
Also we'd probably need to remove any existing icons to avoid double-ups.

But with those caveats in mind, yes I'd be happy for a contributor to take it on! We should require them to manually test it first, to avoid unnecessary back/forth and extra testing.

Reply:
Yes, exactly. I would base it on the target, too.

Reply:
Issue created:

*  #13030